public/main.js

```diff
@@ -96,6 +100,16 @@ function reloadTodoList() {
                                                                 incompleteTodoCount.toString();
     });
 }
+function renderMessageDialog(message, type) {
+    if (messagesDiv.disabled) {
+        messagesDiv.disabled = false;
+    }
+    type = type === undefined ? "error" : type;
```

Comment:
I think es6 has a nice default function parameter mechanism, it would be worth taking a look but leave the code as is.